RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+# Release 2.4.5 2020-03-31
+
+Issue #13402 - reduce test_prs_functions memory footprint so tests run
+
+Issue #13402 - fixed assorted bugs in FDCHP data product algorithms
+
```

Comment:
Please consolidate the 2 release entries into 1

Reply:
done